games-roguelike/angband/angband-4.1.3.ebuild

```diff
@@ -0,0 +1,113 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit autotools eapi7-ver eutils gnome2-utils
+
+MAJOR_PV=$(ver_cut 1-2)
+
+DESCRIPTION="A roguelike dungeon exploration game based on the books of J.R.R. Tolkien"
+HOMEPAGE="https://rephial.org/"
+SRC_URI="https://rephial.org/downloads/${MAJOR_PV}/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="ncurses sdl +sound X"
+
+RDEPEND="X? ( x11-libs/libX11 )
+	!ncurses? ( !X? ( !sdl? ( sys-libs/ncurses:0=[unicode] ) ) )
```

Comment:
Really it baffles me too, but forgive me, I don't want to investigate it right now. This PR was intended as a simple version bump. Let's leave the ebuild improvement and minor bugs fixing for another PR?

Reply:
I got it.

> Is ncurses still always needed?

No, I can make `USE="-ncurses X sdl"` or even `-ncrurses X -sdl -sound`, and it will work, and ncurses won't be required.

Reply:
Yes, I think it was crazy in truth. `REQUIRED_USE` will save us all (I hope).